templates/CRM/common/civicrm.settings.php.template

```diff
@@ -502,6 +502,27 @@ if (CIVICRM_UF === 'UnitTests') {
 //   define('CIVICRM_LOG_ROTATESIZE', 0 );
 // }
 
+/**
+ * Which directories should we exclude when scanning the codebase for things
+ * like extension .info files, or .html partials or .xml files etc. This needs
+ * to be a valid preg_match() pattern.
+ *
+ * If you do not define it, a pattern that excludes dirs starting with a dot is
+ * used, e.g. to exclude .git/). Adding suitable patterns here can vastly speed
+ * up your container rebuilds and cache flushes. The pattern is matched against
+ * the absolute path. Remember to use your system's DIRECTORY_SEPARATOR the
+ * examples below assume /
+ *
+ * Example: This excludes node_modules (can be huge), various CiviCRM dirs that
+ * are unlikely to have anything we need to scan inside, and (what could be
+ * your) Drupal's private file storage area.
+ *
+ * '/(\.|packages|node_modules|bower_components|js/|css/|sites/default/files/private/)'
```

Comment:
I needed to rewrite this example for it to work? `'#(\\\\\.|packages|node_modules|bower_components|js/|css/|sites/default/files/private/)#'`.

Yes 5 backslashes. While backslashes are among the best types of slashes, I would have thought 3 would work.

Anyway the start and end terminators are the main thing.